### PR TITLE
[PLA-187] Use Devise for Administrate

### PR DESCRIPTION
**Pull request by gduquesnay**
… user#role instead

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
-FeedSource.create(feed_type: 'general', name: 'NewsBTC', feed_url: 'https://www.newsbtc.com/feed/', site_url: 'https://www.newsbtc.com/')
-FeedSource.create(feed_type: 'general', name: 'Profit Confidential', feed_url: 'https://www.profitconfidential.com/feed/', site_url: 'https://www.profitconfidential.com/')
-FeedSource.create(feed_type: 'general', name: 'The Bitcoin News', feed_url: 'https://thebitcoinnews.com/feed/', site_url: 'https://thebitcoinnews.com/')
-FeedSource.create(feed_type: 'general', name: 'The Daily Hodl', feed_url: 'https://dailyhodl.com/feed/', site_url: 'https://dailyhodl.com/')
-FeedSource.create(feed_type: 'general', name: 'Trustnodes', feed_url: 'https://www.trustnodes.com/feed', site_url: 'https://www.trustnodes.com/')
-FeedSource.create(feed_type: 'general', name: 'UseTheBitcoin', feed_url: 'https://usethebitcoin.com/feed/', site_url: 'https://usethebitcoin.com/')
+FeedSource.find_or_create_by(feed_type: 'general', name: 'AMBCrypto', feed_url: 'https://ambcrypto.com/feed/', site_url: 'https://ambcrypto.com/')
```
Comment (hanchang) [inline db/seeds.rb]: Hm, IIRC `FeedSource.create` would just fail silently if it already exists; any reason for changing this to `find_or_create_by`?

File: db/seeds.rb [hunk 2]
```diff
@@ -1,39 +1,42 @@
...
+FeedSource.find_or_create_by(feed_type: 'general', name: 'The Bitcoin News', feed_url: 'https://thebitcoinnews.com/feed/', site_url: 'https://thebitcoinnews.com/')
+FeedSource.find_or_create_by(feed_type: 'general', name: 'The Daily Hodl', feed_url: 'https://dailyhodl.com/feed/', site_url: 'https://dailyhodl.com/')
+FeedSource.find_or_create_by(feed_type: 'general', name: 'Trustnodes', feed_url: 'https://www.trustnodes.com/feed', site_url: 'https://www.trustnodes.com/')
+FeedSource.find_or_create_by(feed_type: 'general', name: 'UseTheBitcoin', feed_url: 'https://usethebitcoin.com/feed/', site_url: 'https://usethebitcoin.com/')
 
 #FeedSource.create(feed_type: 'general', name: 'CoinIdol', feed_url: 'https://coinidol.com/rssshort/', site_url: 'https://coinidol.com/') # Removed due to lack of summary and author fields
 #FeedSource.create(feed_type: 'general', name: 'Quartz', feed_url: 'https://qz.com/feed/', site_url: 'https://qz.com/')
+
+admin_emails = ["gduquesnay@gmail.com", "contactdanny19@gmail.com", "han+heap@hanchang.co"]
```
Comment (hanchang) [inline db/seeds.rb]: Let's remove please.

File: db/seeds.rb (hunk 1 above)
Comment (gduquesnay) [reply db/seeds.rb]: Create was failing just not silently.   It was throwing errors like:
PG::UniqueViolation: ERROR:  duplicate key value violates unique constraint "index_feed_sources_on_name"
DETAIL:  Key (name)=(AMBCrypto) already exists.

So changed to find_or_create_by to stop those.

File: db/seeds.rb (hunk 2 above)
Comment (gduquesnay) [reply db/seeds.rb]: Yep.  Will do.

**Pull request merged**